app/external_doc.rb

```diff
@@ -60,7 +61,7 @@ def call
         uri = URI.parse(href)
         path = uri.path
 
-        next if uri.scheme || href.start_with?("#") || path.end_with?(".md")
+        next if uri.scheme || href.start_with?("#")
```

Comment:
Note to self: rewrite this commit message. It should make reference to this change ^